taskproj/employee/models.py

```diff
@@ -0,0 +1,5 @@
+from django.db import models
+
+class Login(models.Model):
```

Comment:
@Ali-Ad This should be named to `User` not `Login` also see base user model in django